jets/e/secp.c

```diff
@@ -0,0 +1,356 @@
+/* j/5/secp.c
+**
+*/
+#include "all.h"
+#include "../include/secp256k1.h"
+#include "../include/secp256k1_recovery.h"
+
+// end-to-end test:
+//   =hash 123
+//   =priv 99
+//   =nonce (make-k:secp256k1:secp:crypto `@uvI`hash priv)
+//   =sig (ecdsa-raw-sign:secp256k1:secp:crypto `@uvI`hash priv)
+//   =reco (ecdsa-raw-recover:secp256k1:secp:crypto `@uvI`hash v:sig  r:sig s:sig)
+
+
+
+
+// util funcs
+//
+
+#if 0
+static void pretty_print_hex(char * headline,
+                             c3_y * ptr,
+                             c3_w num_bytes)
+{
+  fprintf(stderr, "\r%s:\n\r  ", headline);
+  for (c3_ws ii=0 ; ii < num_bytes; ii++) {
+    fprintf(stderr, "%02x", ptr[ii]);
+    if (0 == (ii + 1) % 16){
+          fprintf(stderr, "\n\r  ");
+    } else {
+          fprintf(stderr, ", ");
+    }
+  }
+  fprintf(stderr, "\n\r");
+
+}
+#endif
+
+// no guarantees if 'in' and 'out' overlap / are the same
+static void byte_reverse(c3_y *in,
+                         c3_y *out,
+                         size_t n_bytes)
+{
+  size_t ii;
+  for (ii = 0; ii < n_bytes; ii++){
```

Comment:
Pretty sure you can safely find and replace `){` with `) {`.

Reply:
is that house style?

ok.  search-and-replace

Reply:
done